deployment/streaming/config-deployment.mk.template

```diff
@@ -28,9 +33,12 @@ export NETWORK_CONFIGURATION="{\"awsvpcConfiguration\": {\"subnets\": [\"${ECS_S
 ################################################################################
 #                                                     Batch resource definitions
 ################################################################################
-export SERVICE_ACCESS_SECURITY_GROUP := ${ECS_SECURITY_GROUP}
-export EMR_MASTER_SECURITY_GROUP :=
-export EMR_SLAVE_SECURITY_GROUP :=
+export MASTER_SECURITY_GROUP :=
```

Comment:
Sort of the same question here, looks like a rename. Just double checking that this is fixing up a var name to the correct value rather than accidentally introducing an error.